nilmtk/results.py

```diff
@@ -105,7 +105,7 @@ class as self) from new chunk of data.
         if new_result._data.empty:
             return
 
-        self._data = self._data.append(new_result._data, sort=False)
+        self._data = pd.concat([self._data ,new_result._data], sort=False)
```

Comment:
Style: the space should be after the comma and not before.

Reply:
oh , what should i do?

Reply:
Let me close this pull request

Reply:
@prince7003 you didn't need to close the PR. You just needed to make an additional commit to fix the styling. Feel free to discuss these in person with me. I'm reopening this PR.

Reply:
ok , let me do additional commit

Reply:
done

Reply:
Great! For your information, you can use git to modify your last commit.
```
# Make your changes
git add the_file_that_you_changed.py
git commit --amend  # will change your last commit rather than create a new one.
git push --force-with-lease  # re-write the last commit on the remote server.
```

Reply:
For now, it looks good, and I will merge your PR by concatenating the two commits into one. You have nothing to do.